mkdocs.yml

```diff
@@ -199,8 +199,10 @@ nav:
   - Nautobot Apps:
       - Overview: "apps/index.md"
       - Network to Code App Docs:
+          - Nautobot ChatOps: "https://docs.nautobot.com/projects/chatops/en/latest/"
+          - Nautobot Firewall Models: "https://nautobot-plugin-firewall-models.readthedocs.io/en/latest/"
+          - Nautobot Plugin Nornir: "https://docs.nautobot.com/projects/plugin-nornir/en/latest/"
```

Comment:
Should this be `Nautobot Nornir Plugin` instead? `Nautobot Plugin Nornir` reads awkwardly to me. I know we need to distinguish between `nautobot-plugin-nornir` and `nornir-nautobot` though.

Reply:
Unfortunately, this is correct, we documented the faq (the second one here) https://docs.nautobot.com/projects/plugin-nornir/en/latest/user/app_faq/ to explain why.

Reply:
@glennmatthews I just take whatever the plugin calls itself officially :) I agree that the naming in this case is a bit awkward, but I don't think there's much to be done about it right now.